soccer/Processor.cpp

```diff
@@ -713,24 +748,57 @@ void Processor::sendRadioData() {
     }
 
     // Add RadioTx commands for visible robots and apply joystick input
-    for (OurRobot* r : _state.self) {
-        if (r->visible || _manualID == r->shell()) {
+
+    if (!_multipleManual) {
```

Comment:
Same thing as before. Can we generalize the if case and else case into a single generalized case?